CONTRIBUTING.rst

```diff
@@ -20,9 +20,59 @@
 Contributions
 =============
 
-Contributions are welcome and are greatly appreciated! Every little bit helps,
+Contributions to Airflow are welcome and are greatly appreciated! Every little bit helps,
 and credit will always be given.
 
+We would like more people to use, contribute to, and help maintain Airflow.
+These three goals align with the three main membership roles within our community:
+users, contributors, and committers/maintainers.
+
+We would like to instill a sense of ownership and pride among the members of our community
+by encouraging:
+
+- people to use Airflow
+- users to contribute ro Airflow
+- contributors to become committers
+
+You can contribute to Airflow in various ways such as:
+
+- reporting bug
+- suggesting new features and Road Map items
+- submitting Pull Requests (PRs) for bug fixes and new features.
+- attending Airflow meets-up events near you:
+
+  - search for Airflow `Meetup <https://www.meetup.com/>`_
+
+- helping to answer questions on gitter, the dev email list, PRs, and Jiras.
+
+
+Best Ways of Engaging Yourself in the Community
+-----------------------------------------------
+
+There are several ways for members to connect meaningfully in the community.
+These include:
+
+- Sending an email to: `<dev@airflow.incubator.apache.org>`_
+
+  - This is the best way to get a Committer to answer a question.
+
+- Asking a question on `<https://gitter.im/apache/incubator-airflow>`_
```

Comment:
Gitter? I don't think airflow use this any more

Reply:
Oh! I will remove that then.